docs/get-started.md

```diff
@@ -28,9 +28,9 @@ Prefer to jump right in? Try out some of our examples.
 
 Want to master Temporal Workflows? Our guides will bring you up to speed on Temporal development best practices.
 
-- [How to install Temporal](/docs/install-temporal-server)
-- [How to configure Temporal](/docs/configure-temporal-server)
-- [How to filter Workflows](/docs/filter-workflows)
+- [How to install Temporal](/docs/install-temporal-server/)
+- [How to configure Temporal](/docs/configure-temporal-server/)
+- [How to filter Workflows](/docs/filter-workflows/)
```

Comment:
Missing below (line 34): /docs/tctl -> /docs/tctl/

Reply:
Not sure why, but I couldn't post a comment below line 34.... Github was forcing me to add the comment to the old diff when I did that. So I had to put it above...

Reply:
good catch, fixed